src/models/highlightedtext.h

```diff
@@ -29,7 +27,7 @@ class HighlightedText : public QObject
 {
     Q_OBJECT
 public:
-    HighlightedText(KSyntaxHighlighting::Repository* repository, QObject* parent = nullptr);
+    HighlightedText(KSyntaxHighlighting::Repository* repository, int tabWidth, QObject* parent = nullptr);
```

Comment:
why is this passed as a property here, can't we just call updateTabWidth as needed?